src/test/java/com/example/petbuddybackend/service/animal/AnimalServiceIntegrationTest.java

```diff
@@ -0,0 +1,68 @@
+package com.example.petbuddybackend.service.animal;
+
+import com.example.petbuddybackend.config.TestDataConfiguration;
+import com.example.petbuddybackend.entity.amenity.AnimalAmenity;
+import com.example.petbuddybackend.entity.animal.AnimalAttribute;
+import com.example.petbuddybackend.utils.exception.throweable.NotFoundException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@ContextConfiguration(classes = TestDataConfiguration.class)
+public class AnimalServiceIntegrationTest {
```

Comment:
Nie powiedziałbym że to jest test integracyjny

Reply:
jest, bo jeszcze korzysta z bazy :)